Library/Homebrew/formula_auditor.rb

```diff
@@ -415,6 +408,22 @@ def audit_conflicts
       end
     end
 
+    def audit_gcc_dependency
+      return unless @git
+      return unless @core_tap
```

Comment:
Would be good to have something similar for `audit_version` in `resource_auditor.rb` too but can wait for another PR.

Reply:
Do we need it? I [benchmarked](https://github.com/carlocab/workflow-test/actions/runs/2928863489) `brew audit --tap=homebrew/core --only=version`, and it takes under 15 seconds (closer to 10 on Linux with `HOMEBREW_SIMULATE_MACOS_ON_LINUX`).

That's a long time, but not really by that much when doing `--except=version` already takes two minutes.

Reply:
@carlocab Fine with me! In that case, can you remove the `--except=version` in `brew test-bot`? Just want test bot and `brew audit` to be consistent by default here.

Reply:
Actually, it even looks like most of that 10-15 seconds comes for free, as it's already done/cached elsewhere: https://github.com/carlocab/workflow-test/actions/runs/2928925233/attempts/1#summary-8023071550

Reply:
https://github.com/Homebrew/homebrew-test-bot/pull/823